### DISCUSSION TITLE
Morpheus Template Page Missing

### DISCUSSION THREAD

**Discussion by bkroeger** [primefaces/community#1491] [PrimeFaces Templates] [answered]
Hello, we have Primefaces Elite and are using the Morpheus template which we are upgrading to version 6. I noticed it's no longer available from https://www.primefaces.org/layouts/morpheus.  There also appear to be very few templates available now compared to the past.

We are wondering if Morpheus has ended at version 6 and if we want to get upgrades in the future, if we need to use one of the 8 templates listed at https://www.primefaces.org/store/

**Comment by melloware** [comment]
I think so it seems PF seems to be phasing out the older templates... no official word yet from PrimeTek though so hold tight.

**Comment by kwee27** [comment]
Saw this twitter post from 21 December re PrimeFaces Elite:

> [PrimeFaces](https://twitter.com/primefaces)
> [@primefaces](https://twitter.com/primefaces)
> ·
> [Dec 21, 2023](https://twitter.com/primefaces/status/1737756186248687776)
> As of v13, PrimeFaces Elite is sunsetting, 13.x.x releases are now available to the community as Open Source. 🎉
> 
>
>PrimeFaces LTS replacement is coming up to support older releases (e.g., v8) for security and maintenance, just like Elite did.

**Comment by ZeroIXI** [comment]
Older templates have been taken down as maintanance cost grow and sales drop.

**Reply by melloware** [reply, marked as answer]
Please see: https://github.com/orgs/primefaces/discussions/1806

**Comment by tandraschko** [comment]
in general we decided to only maintain following templates, which were released yesterday:

          - "apollo"
          - "atlantis"
          - "avalon"
          - "diamond"
          - "freya"
          - "poseidon"
          - "ultima"
          - "verona"
